tjsendemail.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * @package     Tjreports.Plugins
+ * @subpackage  Plugins,system,plg_system_sendemail
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2020 Techjoomla. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// No direct access.
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Session\Session;
+
+JHtml::_('jquery.token');
+
+$lang = JFactory::getLanguage();
+$lang->load('plg_system_tjsendemail', JPATH_ADMINISTRATOR);
```

Comment:
use this instead, as a class variable

```
/**
	 * Load plugin language file automatically so that it can be used inside component
	 *
	 * @var    boolean
	 * @since  2.1.2
	 */
	protected $autoloadLanguage = true;
```